### PR TITLE
completed week 1 assessment

### PR DESCRIPTION
**Pull request by Lilmcg**

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -7,42 +7,179 @@
...
+
+var temp = 212
+var boilingPt = 212
+if (temp < boilingPt) {
+  console.log (`${temp} is below boiling point`)
+} else if (temp > boilingPt) {
+  console.log (`${temp} is above boiling point`)
+} else if (temp = boilingPt) {
+  console.log (`${temp} is at boiling point`)
+}
```
Comment (WalkerTexas-coder) [inline code-challenges.js:76]: Great work here! very robust setting up code for each condition! 
Luckily next assessment we'll have functions so you won't have to copy and paste it each time!

File: code-challenges.js [hunk 2]
```diff
@@ -7,42 +7,179 @@
...
+// Declare myString1 as the variable is equal to "charlie"
+// Declare splitMyString1 as the variable is equal to split function of myString1
+// Declare reverseMyString1 as the variable is equal to reverse function of splitMyString1
+  // Print join function of reverseMyString1
+
 var myString1 = "charlie"
-var myString2 = "delta"
+var splitMyString1 = myString1.split("")
+var reverseMyString1 = splitMyString1.reverse()
+console.log(reverseMyString1.join(""))
```
Comment (WalkerTexas-coder) [inline code-challenges.js:100]: Well done!

File: code-challenges.js [hunk 3]
```diff
@@ -7,42 +7,179 @@
...
+// Else if numberRefactor1 is greater than numberRefactor2
+   // Print value of numberRefactor1 minus numberRefactor2
+// Else print value of numberRefactor2 minus numberRefactor1
+
 var numberRefactor1 = 42
 var numberRefactor2 = "hello"
-
-// var numberRefactor1 = 27
-// var numberRefactor2 = 24
+if (isNaN (numberRefactor1) || isNaN (numberRefactor2)) {
```
Comment (WalkerTexas-coder) [inline code-challenges.js:161]: Very good use of isNaN!

File: coding-concepts.js [hunk 4]
```diff
@@ -8,44 +8,57 @@
...
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
 // a) Your answer:
-// b) Verify and explain:
+// SATURDAY
+// SUNDAY
 
+// b) Verify and explain:
+// Initial answer was wrong // terminal returned an error message // toUpperCase function ran into a problem attempting to uppercase the array // javascript requires more info before it can perform this function in an array
```
Comment (WalkerTexas-coder) [inline coding-concepts.js:53]: Cool! so the interesting catch here is that .toUpperCase() is a string method and won't work on an array.

File: interview-questions.md [hunk 5]
```diff
@@ -7,60 +7,72 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
   Researched answer:
+  Pair programming started in the 90's. It is a driver-navigator technique involving two programmers that work on the same system. The programmer writing the code is the driver. While the driver enters the code, the navigator reviews the code and communicates a plan of action. Communication is critical in pair programming. It requires constant back-and-forth communication between the programmers. The driver should communicate throughout the coding process to keep the navigator engaged and to avoid the navigator from zoning out. The programmers should switch roles periodically and the workload should be shared so that each participant gains knowledge over the course of the project. Proper pair programming helps to reduce mistakes, allows for brainstorming, and creates accountability. There are different pairing techniques that can be used depending on the task at hand. In the strong style pairing technique, if the driver has an idea they have to pass over the keyboard. In this case the navigator communicates the idea and makes all the decisions.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:51]: Good stuff!

File: interview-questions.md [hunk 6]
```diff
@@ -7,60 +7,72 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+Git is a version control system that allows you to track the changes that programmers make to code. Git is software that is installed in your local system and uses a command line tool, while Github is a hosting service using a web-based user interface. Github works side by side with Git by pulling data from Git. Github provides a vast amount of storage space and a place to access your projects and your Git working directories and repositories. Github provides extra features that allow for the copying of a repository, creation of notifications for a "pull request" and for merging code changes to the main repository. Github also allows users from all over the world to connect with each other.
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: = == === > < >= <=
 
   Researched answer:
+  Equality (==, ===), relational (><, >=, <=), and logical (and, or, not) operators return a Boolean value of true or false.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:22]: I think this answer will do well in an interview!

**Review by WalkerTexas-coder — APPROVED** [review]
Lillian very comprehensive work! You did such a great job with the psuedo code and keeping track of which code blocks we're being impacted. I really appreciate your interview answers. You did a great job for the first assessment and I am looking forward to t he next one. Your PR is approved!

**Comment by Lilmcg** [comment]
Thank you! I appreciate your feedback!

Lillian

**Pull request merged**